velox/dwio/common/Statistics.h

```diff
@@ -380,6 +380,69 @@ class IntegerColumnStatistics : public virtual ColumnStatistics {
   std::optional<int64_t> sum_;
 };
 
+// Statistics for decimal columns. T could be int64_t or int128_t.
+template <typename T>
```

Comment:
If `T` is int64, why not just use `IntegerColumnStatistics`?

Reply:
Updated. `IntegerColumnStatistics` was used instead.